automatic/git.portable/README.md

```diff
@@ -0,0 +1,9 @@
+# [<img src="" height="48" width="48" /> git.portable](https://chocolatey.org/packages/git.portable)
```

Comment:
@majkinetor you may wish to update the img src url here manually, if I remember correctly the icon url in the nuspec file is up to date, so this one won't be updated.